mne/io/eeglab/eeglab.py

```diff
@@ -84,6 +82,8 @@ def _get_info(eeg, montage, eog=()):
             montage = Montage(np.array(pos), ch_names, kind, selection)
     elif isinstance(montage, string_types):
         path = op.dirname(montage)
+    else:  # if eeg.chanlocs is empty, we still need default chan names
+        ch_names = ["Channel " + str(ii) for ii in range(eeg.nbchan)]
```

Comment:
should this follow the standard naming convention of being `EEG XX` instead of `Channel XX`